includes/admin/class-wc-rest-payments-payment-intents-controller.php

```diff
@@ -46,4 +77,286 @@ public function get_payment_intent( $request ) {
 
 		return $this->forward_request( 'get_intent', [ $payment_intent_id ] );
 	}
+
+	/**
+	 * Create a payment intent.
+	 *
+	 * @param WP_REST_Request $request data about the request.
+	 *
+	 * @throws Rest_Request_Exception
+	 */
+	public function create_payment_intent( $request ) {
+		try {
+
+			$order_id = $request->get_param( 'order_id' );
+			$order    = wc_get_order( $order_id );
+			if ( ! $order ) {
+				throw new Rest_Request_Exception( __( 'Order not found', 'woocommerce-payments' ) );
+			}
+
+			$wcpay_server_request = Create_And_Confirm_Intention::create();
+
+			$currency = strtolower( $order->get_currency() );
+			$amount   = WC_Payments_Utils::prepare_amount( $order->get_total(), $currency );
+			$wcpay_server_request->set_currency_code( $currency );
+			$wcpay_server_request->set_amount( $amount );
+
+			$metadata = $this->gateway->get_metadata_from_order( $order, Payment_Type::SINGLE() );
+			$wcpay_server_request->set_metadata( $metadata );
+
+			$wcpay_server_request->set_customer( $request->get_param( 'customer' ) );
+			$wcpay_server_request->set_level3( $this->gateway->get_level3_data_from_order( $order ) );
+			$wcpay_server_request->set_payment_method( $request->get_param( 'payment_method' ) );
+			$wcpay_server_request->set_payment_method_types( [ 'card' ] );
+			$wcpay_server_request->set_off_session( true );
+			$wcpay_server_request->set_capture_method( $this->gateway->get_option( 'manual_capture' ) && ( 'yes' === $this->gateway->get_option( 'manual_capture' ) ) );
+
+			$wcpay_server_request->assign_hook( 'wcpay_create_and_confirm_intent_request_api' );
+			$intent = $wcpay_server_request->send();
+
+			$response = $this->prepare_item_for_response( $intent, $request );
+			return rest_ensure_response( $this->prepare_response_for_collection( $response ) );
+
+		} catch ( \Throwable $e ) {
+			Logger::error( 'Failed to create an intention via REST API: ' . $e );
+			return new WP_Error( 'wcpay_server_error', $e->getMessage(), [ 'status' => 500 ] );
+		}
+	}
+
+
+	/**
+	 * Item schema.
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		return [
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'payment_intent',
+			'type'       => 'object',
+			'properties' => [
+				'id'       => [
+					'description' => __( 'ID for the payment intent.', 'woocommerce-payments' ),
+					'type'        => 'string',
+					'context'     => [ 'view' ],
+				],
+				'amount'   => [
+					'description' => __( 'The amount of the transaction.', 'woocommerce-payments' ),
+					'type'        => 'string',
+					'context'     => [ 'view' ],
+				],
+				'currency' => [
+					'description' => __( 'The currency of the transaction.', 'woocommerce-payments' ),
+					'type'        => 'string',
+					'context'     => [ 'view' ],
+				],
+				'created'  => [
+					'description' => __( 'Timestamp for when the payment intent was created.', 'woocommerce-payments' ),
+					'type'        => 'string',
```

Comment:
Nitpick. You can add` 'format'  => 'date-time'`

Reply:
Its actually a timestamp, which I'll try and fix later.

Reply:
https://github.com/Automattic/woocommerce-payments-meta/issues/286